src/main/java/org/kapott/hbci/dialog/AbstractRawHBCIDialog.java

```diff
@@ -68,7 +69,7 @@ public final HBCIMsgStatus execute(final DialogContext ctx)
             ctx.setRepeat(false);
             
             // Checken, ob ein Neustart noch moeglich ist:
-            if (this.executions.get() > 2)
+            if (this.executions.get() > 2 && (!Feature.PINTAN_DECOUPLED_REFRESH.isEnabled() || KnownReturncode.W3956.searchReturnValue(ctx.getMsgStatus().segStatus.getWarnings()) == null))
```

Comment:
Hier bin ich mir nicht ganz sicher, ob in "KnownReturncode.W3956.searchReturnValue(ctx.getMsgStatus().segStatus" irgendwas NULL sein kann ("getMsgStatus()" oder "segStatus"?

Reply:
Ah, guter Hinweis. Hab nur gesehen, dass `ctx` bereits vorher benutzt wird, aber `getMsgStatus()` noch nicht. Null check ist hinzugefügt.